LayoutTests/TestExpectations

```diff
@@ -3731,10 +3731,6 @@ webkit.org/b/182042 imported/w3c/web-platform-tests/css/css-fonts/font-default-0
 webkit.org/b/182042 imported/w3c/web-platform-tests/css/css-fonts/font-kerning-03.html [ ImageOnlyFailure ]
 webkit.org/b/86071 imported/w3c/web-platform-tests/css/css-fonts/font-kerning-05.html [ ImageOnlyFailure ]
 
-# font-variant-alternates: swash doesn't work
```

Comment:
This change is unrelated to this patch it seems

Reply:
I noticed that those tests were passing locally when they were expected not to, but I guess this has nothing to do with this patch. I'd remove this part when landing.

Reply:
I removed this.